### PR TITLE
Add Task model

### PR DESCRIPTION
**Pull request by arturs105**
Also puts all models in Toggl.Ultrawave.Models namespace

### CODE REVIEW

**Comment by willsb** [comment]
I feel there's too much going on in a single PR. I think this should be split in a PR to move things and a second one to introduce the `Task` model.

**Comment by arturs105** [comment]
I also thought about that, but if i divide into into 2 PRs, then the one with `Task` model will break some async tests (which return `System.Threading.Tasks.Task`). The return types of those tests will need to be changed to fully qualified type names. And then (probably) changed back after the other PR is accepted.

File: Toggl.Ultrawave/Models/Task.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace Toggl.Ultrawave.Models
+{
+    public class Task : ITask
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        [JsonProperty("pid")]
```
Comment (amulware) [inline Toggl.Ultrawave/Models/Task.cs]: Are there no `project_id` and similar fields in the api for tasks?

Comment (arturs105) [inline Toggl.Ultrawave/Models/Task.cs]: Nope, `pid`, `cid` and `uid`: https://github.com/toggl/toggl_api_docs/blob/master/chapters/tasks.md

Comment (arturs105) [inline Toggl.Ultrawave/Models/Task.cs]: Just checked task endpoint - it returns only short versions.

**Pull request merged**